src/content/developers/docs/consensus-mechanisms/poa/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Proof-of-authority (PoA)
+description: An explanation of the proof-of-authority consensus protocol and its role in blockchain ecosystem.
+lang: en
+---
+
+proof-of-authority (PoA) the modified version of proof-of-stake and the most used consensus mechanism in testnet networks, it has several implementations with the most well known implementation clique.
+
+## Prerequisites {#prerequisites}
+
+To better understand this page, we recommend you first read up on [transactions](/developers/docs/transactions/), [blocks](/developers/docs/blocks/), and [consensus mechanisms](/developers/docs/consensus-mechanisms/).
+
+## What is proof-of-authority (POA)? {#what-is-poa}
+
+Proof-of-Authority is a modified version of [proof-of-stake](/developers/docs/consensus-mechanisms/pos/) that is a reputation-based consensus algorithm instead of stake-based mechanism in POS. The term has been introduced for the first time in 2017 by Gavin Wood, and this consensus algorithm has been mostly used by private chains, testnets and local development networks, as it overcomes the need for high quality resources as POW does, and overcomes the scalability issues with POS by having small subset of nodes storing the blockchian and producing blocks.
+
+The Proof-of-Authority simply requires to trust a subset of authorized signers that are set in the [genesis block](/glossary/#genesis-block), they have the same power in all situations as signers, no signer has exclusive privileges than others in the blockchain as you might think(unless new implementation follow this concept is introduced), the idea behind reputation staking is every authorized signer is well-known to everyone through things like KYC, or by having a well known org shouting they will be the only signer in their blockchain, this way if someone did something wrong, it can be detected who and what their identity is, i.e there will be no attacks on the consensus mechanism by anonymous signers.
+
+The Proof-of-Authority has different implementations, but the ethereum standard implementation is clique that implements [eip-225](https://eips.ethereum.org/EIPS/eip-225), it has the most developer-friendly and easy to implement standard that all clients syncing types are supported, unlike a previously used implementation (Aura) that was implemented in openethereum, which fast,light syncing are not supported because smart contracts are used to store the authorized signers, clique instead only reuse obsoleted headers to solve the issues related to voting to add and remove signers.
```

Comment:
(run-on sentence)

> The Proof-of-Authority

-> "Proof-of-authority"

> the ethereum standard implementation

-> "the standard Ethereum implementation"

> is clique that implements eip-225, ...

-> "is **clique**, which implements EIP-225. ..."

> it has the most developer-friendly and easy to implement standard that all clients syncing types are supported

-> "Clique is developer-friendly and an easy-to-implement standard, supporting all client syncing types."

Would cut the rest, since we seem to be getting into things that probably don't belong in the "What is PoA?" header... Would probably move any discussion about the details of implementations to either "How it works" below.

Reply:
```suggestion
Proof-of-authority has different implementations, but the standard Ethereum implementation is **clique**, which implements [EIP-225](https://eips.ethereum.org/EIPS/eip-225). Clique is developer-friendly and an easy-to-implement standard, supporting all client syncing types.
```